package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-redux-tools",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-redux-tools/issues",
-  "version": "6.0.2",
+  "version": "6.2.0-beta.2",
```

Comment:
Why the jump von 6.0.2 to 6.2.0? Not very semver-like.

And personally I wouldn't commit beta versions but only final versions. As in: publish beta versions locally.
Plus `6.2.0-beta.2` is already published on npm as latest... before this PR is merged? 🤨

Reply:
> Plus 6.2.0-beta.2 is already published on npm as latest... before this PR is merged?

This happens if you publish without `--tag beta` -> `npm publish --tag beta`

Reply:
@zeitwidrig is right. Should be `6.1.0`